dascore/io/index/ingest.py

```
@@ -198,31 +201,76 @@ def typed_value(value) -> TypedValue | None:
-        # columns. Coordinate-envelope-shaped names (e.g. "channel_step")
-        # are ordinary attrs; a genuine collision with a coord envelope
-        # column is handled (warn, coord wins) when the flat view is built.
-        if sanitize_attr_name(name) in RESERVED_ATTR_COLUMNS:
+        status = _attr_name_status(name)
+        if status == "reserved":
             msg = (
                 f"Skipping reserved attr name {name!r}; it collides with a "
                 "structural index column. The attr stays on the patch but "
                 "is not queryable through the spool."
             )
             warnings.warn(msg, UserWarning)
```

Ruff (0.15.21)

[warning] No explicit `stacklevel` keyword argument found

Set `stacklevel=2`

(B028)